src/pages/studyView/addChartButton/geneLevelSelection/GeneLevelSelection.tsx

```diff
@@ -124,18 +177,59 @@ export default class GeneLevelSelection extends React.Component<
         return [];
     }
 
+    private get subOptions() {
```

Comment:
no reason to make this a class member.  it's a constant really

Reply:
extracted to constant under the same file